### ISSUE TITLE
add high score

### ISSUE THREAD

**Issue by jojoyeti** [Shildifreak/SmolSnek#2]
show player the maximum length he reached

**Comment by Shildifreak** [comment]
try something like:
```
document.cookie = "" + highscore;
highscore = Number(document.cookie) || 0;
```

**Comment by Shildifreak** [comment]
implemented in b2133b3176f24171055d3983b1d0e093de5565fe

cookies only work properly when the application is loaded via http:// from a server
some browsers including firefox also allow cookies for file:// but chrome doesn't
